client/components/Pedalboards/Pedalboard/Pedalboard.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Button, Card, Image } from 'semantic-ui-react'
+
+const pedalBoard = ({ price, brand, model, description, image }) => {
+  return (
+    <Card>
+      <Card.Content>
+        <Image src={image} />
+        <Card.Header>{brand}</Card.Header>
+        <Card.Meta>{model}</Card.Meta>
+        <Card.Description>
+          <strong>{price}</strong>
+        </Card.Description>
+        <Card.Description>{description.slice(0, 200) + '...'}</Card.Description>
+      </Card.Content>
+      <Card.Content extra>
+        <div className="ui">
+          <Button size="small" basic color="red">
```

Comment:
This button doesn't do anything, so you should remove it. Don't turn in code that doesn't do anything yet. No placeholders.